website/src/index.html

```diff
@@ -0,0 +1,196 @@
+    <link rel="mask-icon" href="./images/resources/safari-pinned-tab.svg" color="#5bbad5">
+    <link rel="shortcut icon" href="./images/resources/favicon.ico">
+    <meta name="msapplication-TileColor" content="#ffffff">
+    <meta name="msapplication-config" content="./images/resources/browserconfig.xml">
+    <meta name="theme-color" content="#9a93c6">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <div id="loader" class="loader"></div>
+        <nav id="navbar" class="header__nav">
+            <ul>
+                <li class=" header__nav__logo">
+                    <a href="#">
+                        <img src="images/resources/timer.svg" alt="Time To Leave Logo">
+                    </a>
+                </li>
+                <li class="header__nav__item header__nav__button">
+                    <a href="https://github.com/thamara/time-to-leave"><img src="./images/github.svg" alt="Github logo"></a>
+                </li>
+                <li class="header__nav__item header__nav__button">
+                    <a href="https://github.com/thamara/time-to-leave/releases/tag/v.1.5.5" class="button--primary">Download</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+
+    <main id="app">
+        <section class="intro">
+            <div class="marketing">
+                <div class="marketing__container js-reveal">
+                    <h1 class="intro__title">
+                        Time to Leave
+                        <img src="images/resources/timer.svg" alt="Time To Leave Logo">
+                    </h1>
+
+                    <p class="marketing__desc">
+                        Time to Leave is a Desktop app for work time management, looking similar to a time-card system.
+                        It’s
+                        different from other software available as it’s very simple, doesn’t require any database
+                        configuration, and focuses on allowing you to control your time. Its premise is to help you
+                        manage
+                        the time you are working and to notify you when it’s time to leave (therefore the name).
+                    </p>
+
+                    <div class="intro__buttons">
+                        <a href="https://github.com/thamara/time-to-leave" class="button--primary">Source code</a>
+
+
+                        <div class="dropdown">
+                            <button id="js-dropdown" class="button--dropdown">Download</button>
+                            <ul>
+                                <li><a href="#">Mac OS X 10.11+</a></li>
+                                <li><a href="#">Windows 7+</a></li>
+                                <li><a href="#">Linux Debian</a></li>
```

Comment:
This will probably be something we will need to change upon each release. We can investigate later if there's some way of linking to the latest release available of each platform.

```suggestion
                                <li><a href="https://github.com/thamara/time-to-leave/releases/download/v.1.5.5/time-to-leave.dmg">Mac OS X 10.11+</a></li>
                                <li><a href="https://github.com/thamara/time-to-leave/releases/download/v.1.5.5/time_to_leave-1.5.5-setup.msi">Windows 7+</a></li>
                                <li><a href="https://github.com/thamara/time-to-leave/releases/download/v.1.5.5/time-to-leave_1.5.5_amd64.deb">Linux Debian</a></li>
                                <li><a href="https://github.com/thamara/time-to-leave/releases/download/v.1.5.5/time-to-leave-1.5.5-1.amd64.rpm">Linux RPM</a></li>
```